ui/package.json

```diff
@@ -34,6 +34,7 @@
     "grunt-regarde": "~0.1.1",
     "grunt-rename": "~0.1.3",
     "grunt-usemin": "~2.2.0",
+    "jpegtran-bin": "^4.0.0",
```

Comment:
Is this required? I didn't need to specify this dependency for the build to go through.

Reply:
Good catch, that was part of my initial debugging, The error message was regarding jpegtran-bin, so I tried this first before realizing it was a nested dependency that was causing the build error. That line can be removed.